release-notes/Cloud/dataminer_services_change_log_2025.md

```diff
@@ -9,6 +9,71 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 12 June 2025 - Enhancements - Various dataminer.services apps - Improved user session management
+
+When a user goes to one of the dataminer.services apps, the app will now try to refresh their session if there is a valid refresh token. Previously, this was not the case. Most of the time, the user would be signed out in such a situation, but sometimes this could result in a situation where they were in the signed in and signed out state at the same time.
+
+The lifetime of both the access token and refresh tokens have been reduced to make these more secure.
+
+When a user is unauthenticated, the URL will stay it  as is, but the user will see an unauthenticated view. This way, users can share and refresh links even when they are signed out.
+
+### 12 June 2025 - Admin - Add user popup, dropdown
+
+This change applies to Admin users.
+Changing the add organization user and add DMS user into a popup.
+Changing the DMS user to a dropdown. This dropdown consists of organization users that are not in the current DMS yet.
```

Comment:
This description is not clear enough; could you change this to a more extensive description with full sentences, so users will easily be able to understand what you mean with this?